doc/rst/source/gallery.rst

```diff
@@ -14,14 +14,19 @@ complex illustration.
 
 .. jinja:: jinja_ctx
 
-   {% for i in range(1, no_of_examples + 1) %}
+   {% for i in range(1, 53) %}
```

Comment:
The variable `no_of_examples` is defined in `doc/rst/conf.py.in`. It should work, no?

Reply:
Forget it. Now I know why you made this change.